salesagility/SuiteCRM
# Fix merge fields not populating with the bean values

Fixes a bug during the Find Duplicates action from Detail View.
Previously, whenever a new field was added from the multi select, the value would show undefined instead of the value from the bean.

## How To Test This

1. Navigate to a record's detail view
2. Click the action menu, Find Duplicates
3. Add any of the available fields and see they display the correct values and not "undefined"

## Types of changes


### Final checklist
